Add nginx pipeline

Gl4di4torRr opened the pull request:
#### What does this PR do?
Adding an nginx example

#### How should this be tested?

```
1. git clone https://github.com/Gl4di4torRr/container-pipelines.git && git checkout add-nginx-pipeline
2. cd container-pipeline/basic-nginx
3. ansible-playbook -i ./.applier/ galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml
4. ansible-playbook -i ./.applier/ galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml -e source_code_url=https://github.com/Gl4di4torRr/container-pipelines.git -e source_code_ref=add-nginx-pipeline
```

#### Is there a relevant Issue open for this?
https://github.com/redhat-cop/containers-quickstarts/issues/169

#### Who would you like to review this?
@etsauer @huddlesj

etsauer commented:
@Gl4di4torRr don't worry about squashing commits... we'll do that on merge.

etsauer commented:
@Gl4di4torRr we've got a new strategy for making these applier inventories more flexible/testable against alternate forks/branches. Rather than making the user update params files, we've started setting ansible vars in our seed-hosts.yml for defaults, and then using `params_from_vars` to use them. This allows us to change the fork/branch we test against just by passing additional arguments to ansible. E.g.

```
ansible-playbook -i .applier/ playbooks/openshift-cluster-seed.yml -e source_code_branch=mybranch
```

Take a look at the example seed hosts below.

https://github.com/redhat-cop/containers-quickstarts/blob/master/jenkins-masters/hygieia-plugin/.applier/group_vars/seed-hosts.yml

Would you be able to refactor to something similar? I need to go back through some of our other pipelines and update them as well.

Gl4di4torRr commented:
@pabrahamsson @etsauer i updated applier and changed the orignal "how to test" section. I added in a readme as well. If you're okay with it, some of the links won't work yet, I can follow up with a MR right after this if we see an issue in the links.

Gl4di4torRr commented on blue-green-spring/Jenkinsfile:
```diff
@@ -196,4 +196,4 @@ def getGroupIdFromPom(pom) {
   def matcher = readFile(pom) =~ '<artifactId>(.+)</artifactId>'
   matcher ? matcher[0][1] : null
-}
+}
```
I think this is from copy/paste from my text editor (sublime).
